### PR TITLE
Unet training with hybrid L1-SSIM loss

### PR DESCRIPTION
**Pull request by MattsonCam**
Use the existing UNet generator for training with the L1-SSIM hybrid loss instead of the ConvNeXt-based model, keep the model output as raw logits without a final sigmoid activation, and perform robust percentile normalization.

### CODE REVIEW

File: training_orchestration.sh [hunk 1]
```diff
@@ -10,8 +10,8 @@ mlflow run . -e train_model \
   --env-manager local \
   --experiment-name "u2os_nuclear_speckle_prediction_dapi_gold" \
   -P dataset=u2os \
-  -P epochs=300 \
-  -P n_trials=5 \
+  -P epochs=200 \
```
Comment (wli51) [inline training_orchestration.sh:13]: LGTM. I observe that UNets on the ALSF data typically converges the first 100 epochs on any brightfield to cell painting prediction with a train sample size of ~3000 256 by 256 images. If you want to save time further you can shorten the patience and the max epoch further.

**Review by wli51 — APPROVED** [review]
Wow single digit diff PR. LGTM!

File: training_orchestration.sh (hunk 1 above)
Comment (MattsonCam) [reply training_orchestration.sh:13]: Sounds good, I'll keep that in mind

**Comment by MattsonCam** [comment]
> Wow single digit diff PR. LGTM!

Yes, trying to stay efficient! Thanks @wli51 ! Merging now
